packages/react/src/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { forwardRef, useMemo } from 'react'
+import type { ForwardedRef, FunctionComponent, HTMLAttributes, PropsWithChildren } from 'react'
+
+function isReactFunctionCom<Tprops>(
+  element: React.ReactNode,
+): element is React.ReactElement<Tprops, FunctionComponent> {
+  return React.isValidElement(element)
+}
+
+interface BreadcrumbComponent
+  extends React.ForwardRefExoticComponent<
+    PropsWithChildren<HTMLAttributes<HTMLElement>> & React.RefAttributes<HTMLElement>
+  > {
+  Item: typeof BreadcrumbItem
+}
+
+// We're validating against an index. Maxiumum children are 8.
+const breadcrumbLimit = 8
```

Comment:
As discussed: for now we won't check children amount

Reply:
done